vagrant/debian/patch.sh

```diff
@@ -28,7 +28,7 @@ set -e
 # Force predictable network device files.
 # Note, a reboot is required to take this changes into account.
 echo "Patching network configuration ..."
-sed --in-place --expression='s/eth0/ens6/' --expression='s/eth2/ens8/' /etc/network/interfaces
+sed --in-place --expression='s/eth0/enp0s3/' --expression='s/eth2/enp0s9/' /etc/network/interfaces
```

Comment:
These device names do not match with the `altnames` i see on my libvirt based box:
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host noprefixroute 
       valid_lft forever preferred_lft forever
2: ens6: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether 52:54:00:33:e6:e6 brd ff:ff:ff:ff:ff:ff
    altname enp0s6
    inet 192.168.121.24/24 metric 100 brd 192.168.121.255 scope global dynamic ens6
       valid_lft 3573sec preferred_lft 3573sec
3: ens7: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 52:54:00:a7:30:5e brd ff:ff:ff:ff:ff:ff
    altname enp0s7
4: ens8: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 52:54:00:3e:3b:30 brd ff:ff:ff:ff:ff:ff
    altname enp0s8
```

Reply:
I made the device names variable per provider.